.github/workflows/publish-extension-nightly.yml

```diff
@@ -49,7 +49,7 @@ jobs:
 
       - name: Set Environment Variables
         run: |
-          echo "DENDRON_RELEASE_VERSION=`cat ./packages/plugin-core/package.json | jq ".version" -r | awk -F. -v OFS=. 'NF==1{print ++$NF}; NF>1{$NF=sprintf("%0*d", length($NF), ($NF+1)); print}'`" >> $GITHUB_ENV
+          echo "DENDRON_RELEASE_VERSION=`(cat ./packages/plugin-core/package.json | jq .version -r; npx vsce show dendron.nightly --json | jq .versions[0].version -r) | sort -rn | head -n 1 | awk -F. -v OFS=. 'NF==1{print ++$NF}; NF>1{$NF=sprintf("%0*d", length($NF), ($NF+1)); print}'`" >> $GITHUB_ENV
```

Comment:
since we aren't using the version in `package.json`, we can simplify this to 
```
echo "DENDRON_RELEASE_VERSION=`(npx vsce show dendron.nightly --json | jq ".versions[0].version" -r) | sort -rn | head -n 1 | awk -F. -v OFS=. 'NF==1{print ++$NF}; NF>1{$NF=sprintf("%0*d", length($NF), ($NF+1)); print}'`" >> $GITHUB_ENV
```

Reply:
We actually need `package.json` for cases when the new major version gets released. E.g. when we release 0.80.0 we still have 0.79.6 as the latest nightly one and we would release 0.79.7.

Reply:
ah, good catch. approved!